packages/cashier/src/components/percentage-selector/percentage-selector.tsx

```diff
@@ -3,30 +3,33 @@ import { Text } from '@deriv/components';
 import { formatMoney, getCurrencyDisplayCode, getDecimalPlaces } from '@deriv/shared';
 import { Localize } from '@deriv/translations';
 import { TReactMouseEvent } from '../../types';
+import { useExchangeRate } from '@deriv/hooks';
 
 type TPercentageSelectorProps = {
     amount: number;
-    currency: string;
     from_account?: string;
-    getCalculatedAmount: (amount: string) => void;
+    getCalculatedAmount: (amount: string, exchanged_rate: number) => void;
     percentage: number;
     should_percentage_reset: boolean;
     to_account?: string;
+    from_currency: string;
+    to_currency: string;
 };
 
 type TCalculateAmountInputEvent = { target: { id: number } };
 
 const PercentageSelector = ({
     amount,
-    currency,
     from_account,
     getCalculatedAmount,
     percentage,
     should_percentage_reset,
     to_account,
+    from_currency,
+    to_currency,
 }: TPercentageSelectorProps) => {
     const [selected_percentage, setSelectedPercentage] = React.useState<number | string>('0');
-
+    const { getRate } = useExchangeRate();
```

Comment:
We are coupling `exchangeRate` in this `PercentageSelector` component which will make the component unusable if it's needed for some other use case that doesn't require exchangeRate.

Reply:
@balakrishna-deriv This should be fine for this PR purpose 🤔 

We are currently refactoring cashier and hopefully all these components are going to be removed or rewrite into reusable ones.

Reply:
I am fine if you are okay with this 😄